scss/detail.scss

```diff
@@ -0,0 +1,48 @@
+@import 'variables';
+
+.text{
```

Comment:
Nice! But we can improve this, right now you can extend this class to the others, but with more classes you'll always have to extend it.

Have you tried using a diferent selector instead of `.text`? Like * or body

Reply:
i think i get it, i could make all the elements inside the body to have certain format and i wont have to use the extend since everything that lives inside the body will automatically have that format right??

Reply:
That's right! And that rules applied to the body can be overridden in the future if you need another font for the titles for example